<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from /home/chayanin09/FUN4_6509/src/robot_control/urdf/my-robot.xacro | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="example">
  <link name="link_0">
    <visual>
      <geometry>
        <mesh filename="file:////home/chayanin09/FUN4_6509/install/robot_control/share/robot_control/meshes/link_0.stl"/>
      </geometry>
    </visual>
  </link>
  <link name="link_1">
    <visual>
      <geometry>
        <mesh filename="file:////home/chayanin09/FUN4_6509/install/robot_control/share/robot_control/meshes/link_1.stl"/>
      </geometry>
    </visual>
  </link>
  <joint name="joint_1" type="revolute">
    <parent link="link_0"/>
    <child link="link_1"/>
    <origin rpy="0 0 0" xyz="0 0 0.2"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="-3.1459" upper="3.1459" velocity="1000.0"/>
  </joint>
  <link name="link_2">
    <visual>
      <origin rpy="0 0 -1.57" xyz="0.0 0.0 0.0"/>
      <geometry>
        <mesh filename="file:////home/chayanin09/FUN4_6509/install/robot_control/share/robot_control/meshes/link_2.stl"/>
      </geometry>
    </visual>
  </link>
  <joint name="joint_2" type="revolute">
    <parent link="link_1"/>
    <child link="link_2"/>
    <origin rpy="-1.57 0 0" xyz="0.0 -0.12 0.0"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="-3.1459" upper="3.1459" velocity="1000.0"/>
  </joint>
  <link name="link_3">
    <visual>
      <origin rpy="0 0 -1.57" xyz="0.0 0.0 0.0"/>
      <geometry>
        <mesh filename="file:////home/chayanin09/FUN4_6509/install/robot_control/share/robot_control/meshes/link_3.stl"/>
      </geometry>
    </visual>
  </link>
  <joint name="joint_3" type="revolute">
    <parent link="link_2"/>
    <child link="link_3"/>
    <origin rpy="0 0 0" xyz="0.0 -0.25 0.1"/>
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="-3.1459" upper="3.1459" velocity="1000.0"/>
  </joint>
  <link name="end_effector">
    <visual>
      <origin rpy="0 0 0" xyz="0.0 0.0 0.0"/>
      <geometry>
        <mesh filename="file:////home/chayanin09/FUN4_6509/install/robot_control/share/robot_control/meshes/end_effector.stl"/>
      </geometry>
    </visual>
  </link>
  <joint name="joint_end_effector" type="fixed">
    <parent link="link_3"/>
    <child link="end_effector"/>
    <origin rpy="1.57 0 0" xyz="0.0 -0.28 0.0"/>
  </joint>
</robot>
